src/v2/Apps/Artist/Routes/WorksForSale/Components/ZeroState.tsx

```diff
@@ -1,31 +1,24 @@
-import { ContextModule } from "@artsy/cohesion"
-import { Clickable, Flex, Message, Text } from "@artsy/palette"
-import { FollowArtistButtonFragmentContainer as FollowArtistButton } from "v2/Components/FollowButton/FollowArtistButton"
-
-export const ZeroState = props => {
-  const { isFollowed, artist } = props
+import { Box, Message, Text } from "@artsy/palette"
+import { Sticky } from "v2/Components/Sticky"
 
+export const ZeroState: React.FC = () => {
   return (
-    <Message>
-      There aren’t any works available by the artist at this time.{" "}
-      {!isFollowed && (
-        <Flex>
-          <FollowArtistButton
-            artist={artist}
-            contextModule={ContextModule.worksForSaleRail}
-            render={({ name }) => {
-              return (
-                <Clickable cursor="pointer" textDecoration="underline">
-                  <Text>Follow {name}</Text>
-                </Clickable>
-              )
-            }}
-          />{" "}
-          <Text pl={0.5}>
-            to receive notifications when new works are added.
-          </Text>
-        </Flex>
-      )}
-    </Message>
+    <Box width="100%" my={1}>
+      <Sticky>
+        {({ stuck }) => {
+          return (
+            <Box pt={stuck ? 1 : 0}>
+              <Message>
+                <Text>No works available by the artist at this time</Text>
+                <Text textColor="black60">
+                  Create an Alert to receive notifications when new works are
+                  added
+                </Text>
+              </Message>
+            </Box>
+          )
+        }}
+      </Sticky>
```

Comment:
**question**: is it necessary to make this block sticky? Because of this, undesirable behavior occurs

https://user-images.githubusercontent.com/3513494/169303487-d7e82e1f-edaf-4418-9279-e2cc1df6e10a.mp4

Reply:
@dimatretyak I would say yes, it was implemented on [this pr](https://github.com/artsy/force/pull/10070).

 I was thinking that maybe we need to report that to the grow team and they will handle it because if you visit artsy.net you can replicate this behaviour on a filtered artwork grid without artworks but it only happens in some resolutions

Reply:
cc @dzucconi @damassi do you want us to create a ticket for [this](https://github.com/artsy/force/pull/10062#discussion_r877049732) ?

Reply:
Yeah, make a ticket. We have the ability to bound it to the box there so may as well.